performance-tests/bench/report_parser/argument_parser.cpp

```diff
@@ -0,0 +1,136 @@
+#include "argument_parser.h"
```

Comment:
Let's make the header / cpp name match the class name it contains, especially when there's only one class defined. This goes for the formatters as well.

Reply:
Done.